firebase-config/src/main/java/com/google/firebase/remoteconfig/FirebaseRemoteConfig.java

```diff
@@ -324,10 +326,10 @@ public boolean activateFetched() {
    * <p>Depending on the time elapsed since the last fetch from the Firebase Remote Config backend,
    * configs are either served from local storage, or fetched from the backend.
    *
-   * <p>To identify the current app instance, the fetch request creates a Firebase Instance ID
-   * token, which periodically sends data to the Firebase backend. To stop the periodic sync, call
-   * {@link com.google.firebase.iid.FirebaseInstanceId#deleteInstanceId}. To create a new token and
-   * resume the periodic sync, call {@code fetchConfig} again.
+   * <p>To identify the current app instance, the fetch request creates a Firebase installation ID,
+   * which periodically sends data to the Firebase backend. To stop the periodic sync, call {@link
+   * com.google.firebase.installations.FirebaseInstallations#delete}. To create a new token and
+   * resume the periodic sync, call {@code fetch} again.
```

Comment:
How about removing this duplicated Javadoc and instead refer to the other method, e.g. "For documentation please refer to {@link fetch()}."

Reply:
@egilmorez What do you think? Is there a general practice for duplicated docs?

Reply:
Chatted about this with @rachelsaunders. Generally you want to avoid linking between methods with specific implementation details since it requires the user to parse out what does/doesn't apply to that method.